lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -31,9 +31,9 @@
 %>
 
 % if cert_status['status'] != 'processing':
-  <div class="message message-status ${status_css_class} is-shown">
-    % if not cert_status['certificate_message_viewable']:
-      % if (cert_status['status'] == 'generating' or cert_status['status'] == 'downloadable'):
+  % if not cert_status['certificate_message_viewable']:
+    % if (cert_status['status'] == 'generating' or cert_status['status'] == 'downloadable') and course_overview.certificate_available_date:
```

Comment:
Not a release blocker, but:
For code readability and maintenance, can you either (1) add a helper method for this entire precondition check, or (2) add it as a new field in `cert_status`?  In general, especially for the certificate code, it's better to explicitly name each of these conditional states that are being checked - for example, by creating a new helper method in the single business-logic python module.  I can explain further in person if this message is cryptic via github.

Reply:
Would you be alright with me doing this in a separate PR? There are a few cases in this template that do this so I might as well clean those up, as well.

Reply:
yes, of course.  thanks for considering.  :)

Reply:
Primary goal is to get the release fixed. Worry about cleanup later.

Reply:
Oops. I misread one of the comments as saying this should be done now. Carry on. Sorry.

Reply:
yes, agree.

Reply:
Merged :)